src/components/RecruitmentProcess/index.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { Fragment } from 'react';
+import moment from 'moment';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import Checkbox from '@mui/material/Checkbox';
+import Button from '@mui/material/Button';
+import { StyledTableCell, StyledTableRow } from '../TableStyling';

```

Comment:
If you're just overriding styles, you can put everything in the same folder a create a file where you can put these styles

Reply:
Oh this I'm talking about line 12 🙄

Reply:
Ok that's what I did